index.html

```diff
@@ -976,7 +960,6 @@ <h2>GeoMundus History</h2>
 	<script src="assets/js/jquery.scrolly.min.js"></script>
 	<script src="assets/js/skel.min.js"></script>
 	<script src="assets/js/util.js"></script>
-	<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
```

Comment:
Why did you delete this line, it's not a comment 
Check (https://en.wikipedia.org/wiki/Conditional_comment)
please restore it